examples/testmonitor/steps.py

```diff
@@ -44,7 +46,7 @@ def create_test_result():
 create_response = create_test_result()
 
 # Create the step requests
-result_id = create_response.results[0].result_id
+result_id = cast(str, create_response.results[0].id)
```

Comment:
Note: The `Result` model doesn't have a `result_id` property, but it does have an `id` property that is marked Optional (presumably because of projections).